client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -6,18 +6,17 @@ import useStyles from './useStyles';
 interface Props {
   linkTo: string;
   btnText: string;
+  variant: any;
```

Comment:
Avoid using any as type, please refer to MUI docs for variant type and update accordingly

Reply:
I've tried string, however it will cause an error on variant={variant}.